pandas/core/indexes/interval.py

```diff
@@ -399,7 +399,8 @@ def inferred_type(self) -> str:
         """Return a string of the type inferred from the values"""
         return "interval"
 
-    @Appender(Index.memory_usage.__doc__)
+    # Cannot determine type of "memory_usage"
+    @Appender(Index.memory_usage.__doc__)  # type: ignore[has-type]
```

Comment:
How is this related?

Reply:
This seems like a mypy bug. 

Adding 
```
from pandas._libs.internals import BlockValuesRefs
```

to the base file causes this